test/models/subject_test.rb

```diff
@@ -83,4 +83,214 @@ class SubjectTest < ActiveSupport::TestCase
     Subject.sync(remote_subject)
     assert_equal 'repository full name', Subject.last.repository_full_name
   end
+
+  test 'sync sets the github_id' do
+    remote_subject = Oj.load(File.open(file_fixture('subject_56.json')))
```

Comment:
Opening and parsing the fixture feels like it could be a useful test helper method to extract?

Reply:
Sounds good! That should be addressed in d08d565